.github/workflows/lantern.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           # when changing supported versions here, please modify supported versions
           # in install.R
           - {os: macOS, version: cpu-intel, runner: macos-13}
-          - {os: macOS, version: cpu-m1, runner: [self-hosted, m1]}
+          - {os: macOS, version: cpu-m1, runner: macos-13}
```

Comment:
I think you actually want `macos-14`, AFAICT macos-13 runs on intel macos runners. See the table in: https://github.com/actions/runner-images/tree/main?tab=readme-ov-file#available-images

Reply:
thanks!